src/Npgsql/Schema/DbColumnSchemaGenerator.cs

```diff
@@ -150,10 +150,8 @@ internal ReadOnlyCollection<NpgsqlDbColumn> GetColumnSchema()
                     populatedColumns++;
                 }
 
-                var fieldName = field.Name.StartsWith("?column?") ? null : field.Name;
-
-                column.BaseColumnName ??= fieldName;
-                column.ColumnName = fieldName;
+                column.ColumnName = field.Name.StartsWith("?column?") ? null : field.Name;
+                column.IsAliased = column.BaseColumnName is null ? default(bool?) : (column.BaseColumnName != column.ColumnName);
```

Comment:
This no longer seems to be setting BaseColumnName anywhere, is that intentional?

Reply:
Not true, there is one place where `BaseColumnName` gets specified.

Reply:
My bad, you're right.